code/handlers/trackservers.q

```diff
@@ -175,13 +173,16 @@ retrydiscovery:{
 // Called by the discovery service when it restarts
 autodiscovery:{if[DISCOVERYRETRY>0; .servers.retrydiscovery[]]}
 
+//check if .proc.getattributes exists in the nontorqprocess
+checknontorqattr:{(`proc in key`)&(`getattributes in key`.proc)};
```

Comment:
Is the first condition even necessary?

```q
q).proc
'.proc
  [0]  .proc
       ^
q)key`.proc
q)`getattributes in key`.proc
0b
```
Is there a condition under which this is insufficient?

Also, even if it is necessary, parentheses around second condition can be removed